config/salesforce/fields.yml

```diff
@@ -1,290 +1,291 @@
-field_update_comment:
-  index_fields:
-    Application__c:
-    Processing_Comment__c:
-    Processing_Date_Updated__c:
-    Processing_Status__c:
+# field_update_comment:
```

Comment:
@fddayan why is this file commented out?

Reply:
I splitted the files and moved each individual section to `saleforce/fields/` folder. Actually, I have to remove this file